packages/input/src/presenters/InputPresenter.js

```diff
@@ -29,11 +29,15 @@ export default function InputPresenter(props) {
           resolvedRoles
         );
         const cssStyles = customStylesheet
-          ? customStylesheet(styles, props, resolvedRoles, metadata.colorSchemeId)
```

Comment:
This change is bundled in a commit that touches text-area, did you want to isolate this?

Reply:
good catch, will make sure to get this out of here.